config/StoredProcedures/CreateUser.sql

```diff
@@ -10,8 +8,18 @@ BEGIN
 
 DECLARE pmid INT;
 select @pmid:=min(PermsID) from Permissions;
+select @unique:= count(name) from
+(select Username as name from Users
+union
+select GroupName as name from UserGroups) as s1 where s1.name=_Username;
+
+IF @unique<=0 THEN
+	INSERT INTO Users VALUES(_Username,_FirstName,_LastName,_Email,@pmid,_HP,_US,_DateCreated);
```

Comment:
Here, between Users and VALUES, can we add (Username, FirstName, LastName, Email, PermsID, HP, US, DateCreated) please. Since we arent inserting a value for isConfirmed, this insert is likely to fail since we arent explicitly stating which columns we are inserting into.